packages/frameworks-sveltekit/src/lib/index.ts

```diff
@@ -202,7 +202,7 @@ export async function getSession(
   config.secret ??= env.AUTH_SECRET
   config.trustHost ??= true
 
-  const url = new URL("/api/auth/session", req.url)
+  const url = new URL("/auth/session", req.url)
```

Comment:
Could you replace the type  of config to `SvelteKitAuthConfig` and use the `prefix`:

```suggestion
  const prefix = config.prefix ?? "/auth"
  const url = new URL(prefix + "/session", req.url)
```